internal/support/env.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+package support
+
+import (
+	"google.golang.org/protobuf/proto"
+	"namespacelabs.dev/foundation/internal/fnerrors"
+	"namespacelabs.dev/foundation/schema"
+)
+
+func MergeEnvs(base []*schema.BinaryConfig_EnvEntry, additional []*schema.BinaryConfig_EnvEntry) ([]*schema.BinaryConfig_EnvEntry, error) {
+	merged := append(base, additional...)
+
+	// Check for collisions.
+	cache := make(map[string]*schema.BinaryConfig_EnvEntry)
+	for _, entry := range merged {
```

Comment:
Isn't this the same implementation as parsing/env.go?

Reply:
Was supposed to be a move.
Can't live here due to dep cycle. Cleaned up.